### PR TITLE
Feat/cc 1159/upgrade to v20

### PR DESCRIPTION
**Pull request by tobafolaranmi-ch**

### CODE REVIEW

File: terraform/groups/ecs-service/variables.tf [hunk 1]
```diff
@@ -57,107 +35,91 @@ variable "desired_task_count" {
...
-variable "session_create" {
-  type        = string
-  default     = "off"
+  description = "The log level for services to use: trace, debug, info or error"
 }
-variable "show_service_unavailable_page" {
-  type        = string
-  default     = "off"
+
+variable "cookie_name" {
```
Comment (harishkm2k) [inline terraform/groups/ecs-service/variables.tf:118]: Do we need this terraform variable? This seems not used anywhere. Isn't it?

File: terraform/groups/ecs-service/locals.tf [hunk 2]
```diff
@@ -2,81 +2,66 @@
...
+  ssm_service_version_map = [
+    for sec in module.secrets.secrets : {
+      name = "${replace(upper(local.service_name), "-", "_")}_${var.ssm_version_prefix}${replace(upper(basename(sec.name)), "-", "_")}", value = sec.version
+    }
   ]
+
+  task_secrets = concat(local.global_secret_list,local.service_secret_list,[])
+
+  task_environment = concat(local.ssm_global_version_map,local.ssm_service_version_map,[
+    { "name" : "EXTENSIONS_WEB_VERSION", "value" : "${var.extensions_web_version}" }
```
Comment (harishkm2k) [inline terraform/groups/ecs-service/locals.tf:64]: Is this variable to be passed to the Application? There is a variable defined. Not having a default value for it.

Comment (harishkm2k) [reply terraform/groups/ecs-service/locals.tf:64]: Usually this is variable used when running the terraform only.
- task: set-terraform-variables
            $(cat github-release/tag)
            
Not sure if the web application needs this value?

Comment (tobafolaranmi-ch) [reply terraform/groups/ecs-service/locals.tf:64]: it is needed ```          cat <<EOF >> terraform-variables/variables.tfvars
          docker_registry=((secrets.shared-services-eu-west-2-docker-registry))
          docker_repo=extensions-web
          extensions_web_version=$(cat github-release/tag)```

File: terraform/groups/ecs-service/variables.tf (hunk 1 above)
Comment (tobafolaranmi-ch) [reply terraform/groups/ecs-service/variables.tf]: removed

**Review by harishkm2k — APPROVED** [review]

**Pull request merged**